Makefile

```diff
@@ -316,6 +319,9 @@ fullinstalljenkins: cleanjenkins alldeps_private buildjenkins installjenkins
 # for nonccl cuda8 build
 fullinstalljenkins2: cleanjenkins alldeps_private2 buildjenkins installjenkins
 	mkdir -p src/interface_py/dist2/ && mv src/interface_py/dist/*.whl src/interface_py/dist2/
+# for nccl cuda9 build
+fullinstalljenkins4: cleanjenkins alldeps_private buildjenkins installjenkins
+	mkdir -p src/interface_py/dist4/ && mv src/interface_py/dist/*.whl src/interface_py/dist4/
 # for nonccl cuda9 build
 fullinstalljenkins3: cleanjenkins alldeps_private2 buildjenkins installjenkins
 	mkdir -p src/interface_py/dist3/ && mv src/interface_py/dist/*.whl src/interface_py/dist3/
```

Comment:
What does `fullinstalljenkins*` imply in terms of the number system used?

Reply:
Ya, all the names can be changed once things actually work.  Note that the names don't ensure anything, since what was done previously to this PR had names wrong (there was a cuda8 instead of a cuda9).  Having a loop over stages and select from a list is best, but someone else can pick that up.

Reply:
Also, alot of this going to be deprecated by my changes to the h2oai branch of our xgboost.  It already uploads 4 xgboost wheels, so we don't need all 4 cases for fullinstalljenkins.